File: src/sdk-reference/csharp/1/document/delete/snippets/delete.cs
try {
  string id = kuzzle.document.delete(
    "nyc-open-data",
    "yellow-taxi",
    "some-id");

  Console.WriteLine("Document " + id + " successfully deleted");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}

File: src/sdk/java/1/controllers/document/delete/snippets/delete.java
try {
    String id = kuzzle.getDocument().delete("nyc-open-data", "yellow-taxi", "some-id");

    if (id.equals("some-id")) {
        System.out.println("Success");
    }
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
